rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -161,13 +163,20 @@ <h3>
       </nav>
     </header>
     <article class="tab-content">
+      {% if holdings_count > 0 %}
       <section class="tab-pane show active p-4" id="documents-get" role="tabpanel" aria-labelledby="documents-get-tab">
-        {%- if record | document_main_type == 'docmaintype_book' %}
-          {% include('rero_ils/_documents_get_book.html') %}
-        {%- else %}
-          {% include('rero_ils/_documents_get.html') %}
-        {%- endif %}
+        {% include('rero_ils/_anonymous_button.html') %}
+        {% if record.harvested %}
+          {% include('rero_ils/_document_online.html') %}
+        {% else %}
+          <public-holdings-items
+            documenttype="{{ record.type[0].main_type }}"
```

Comment:
There is a filter for this `{{ record | | document_main_type }}`

Reply:
done